source/WindowScroller/WindowScroller.js

```diff
@@ -127,6 +131,8 @@ export default class WindowScroller extends PureComponent {
 
   // Referenced by utils/onScroll
   __handleWindowScrollEvent (event) {
+    if (!this._isMounted) return
```

Comment:
@mking-clari Good point, though looking at the code and playing around with it I couldn't pin point why it isn't accurate. New WindowScrollers are added to `mountedInstances` array on mount and are removed on unmount. My best guess is that there's a race condition since scroll events are fired rapidly. If that's the case, I'm not sure what the best course of action is, perhaps `debounce` the scroll event handler?

Alternatively, I can move the check to see if the instance is mounted to the `onScroll.js` helper, changing line 47 to:

```js
if (instance._isMounted && instance.scrollElement === event.currentTarget) {
```

though that still requires using a flag. Any other thoughts?